mitmproxy/tools/console/keymap.py

```diff
@@ -71,9 +72,7 @@ def sortkey(self):
 class Keymap:
     def __init__(self, master):
         self.executor = commandexecutor.CommandExecutor(master)
-        self.keys: dict[str, dict[str, Binding]] = {}
-        for c in Contexts:
-            self.keys[c] = {}
+        self.keys: dict[str, dict[str, Binding]] = defaultdict(dict)
```

Comment:
What is this change about?

Reply:
I thought it'd make it more readable. Should I revert this?

Reply:
Looking at bit closer this is probably fine :)